vcpkg/ports/qgis/bigobj.patch

```diff
@@ -2,11 +2,11 @@ diff --git a/CMakeLists.txt b/CMakeLists.txt
 index 9d48e77a31..e360e29e42 100644
```

Comment:
Are we able to drop this patch (separate PR / unrelated)?

https://github.com/qgis/QGIS/pull/59289